app/tools/android/make_apk_test.py

```diff
@@ -781,6 +781,50 @@ def testManifestWithError(self):
     self.assertTrue(
         out.find('\'Telephony\' related API is not supported') != -1)
 
+  def testManifestWithHtmlFile(self):
+    manifest_path = os.path.join('test_data', 'manifest',
+                                 'unite', 'manifest.json')
```

Comment:
I don't understand why this new directory is called "unite". What is it supposed to mean? Can you come up with a more descriptive name, like "manifest_and_app_root", or "cli_parser_priority"?